tests/TestUtilities.py

```diff
@@ -46,6 +47,7 @@
     'JavaScript Object Notation',
     'JavaServer Pages',
     'Jinja2',
+    'KornShell',
```

Comment:
Not needed.

Reply:
SEE THIS IS ALREADY PRESENT https://github.com/coala/coala/blob/master/tests/TestUtilities.py
It is comparing wrong  PLEASE SEE https://github.com/coala/coala/pull/5748/commits/01a27859071b51ffcd783eb3b283564b8f130bdb

Reply:
It's because you've used wrong ```git``` commands. 
It looks like that you've messed up your branch, I would suggest you to reset to the old commit and then make the changes once again.